handbook/product/beta_and_experimental_feature_labels.md

```diff
@@ -0,0 +1,61 @@
+# Beta and experimental feature labels
+
+We use a hierarchy of feature labels to set expectations with our users and internally: `Experimental` << `Beta` << GA. 
+
+GA ("General Availability") has no in-product label and is the default user assumption for an unlabelled feature. 
+
+## Why we use these labels:
+
+Labelling features correctly lets us move quickly and release early in order to validate our direction. An `Experimental` or `Beta` label and its [corresponding user-facing documentation](https://docs.sourcegraph.com/admin/beta_and_prototype_features): 
+  
+  - Set expectations so users are more forgiving of bugs
+  - Make users less likely to assume the overall quality of Sourcegraph is as brittle as an early feature might be
+  - Communicate that we are particularly interested in user feedback
+  - Communicate that a user's feedback is more likely to have meaningful impact due to the early stage
+  - Let us release paid-tier features initially for free, to get usage feedback fast or to build our sales and marketing knowledge
+
+## When to use each label: 
+
+There's no requirement that every feature begin as `Experimental` and pass through `Beta` before being in `GA`. This depends on the feature size, the development time, and the cost of getting a GA release wrong. 
+
+> While there is no strict required lifecycle, for the sake of our customers and our customer-facing teams, we **do** adhere to the minimum documentation, feature flag, and support requirements [defined in our docs](https://docs.sourcegraph.com/admin/beta_and_prototype_features). 
+
+> For example, you can't label a feature that's enabled by default `Experimental`, nor a feature with no documentation a `Beta`, but you can have the full necessary for `Beta` and still choose to use `Experimental`. 
```

Comment:
API Docs are enabled on sourcegraph.com by default and is labeled as Experimental which violates this example 🤔

Reply:
@jeanduplessis that'd actually be a bit "worse" than violating this example – it violates our live product docs https://docs.sourcegraph.com/admin/beta_and_prototype_features. 

But I'm not sure that's actually the case here. Are API Docs a cloud-only feature? If so, perhaps that docs page should except cloud features. If they're enabled by default on customer instances, then yes, I think we need to decide why that is (are they beta-level? If not, why are they enabled by default? Does that mean we need to reexamine the logic of this page?)